src/corvus.c

```diff
@@ -340,6 +351,21 @@ void start_worker(int i)
 int main(int argc, const char *argv[])
 {
     int i, err;
+
+    /* Load libgcc early to safely call backtrace in signal handler.
+     *
+     * See:
+     * https://www.scribd.com/doc/3726406/Crash-N-Burn-Writing-Linux-application-fault-handlers
+     * https://github.com/gby/libcrash/blob/master/crash.c#L279
+     */
+    void *dummy_trace[1];
+    backtrace(dummy_trace, 1);
```

Comment:
Is [another information source](https://github.com/gby/libcrash "Writing Linux application fault handlers") also useful for this development approach?